tsconfig.json

```diff
@@ -20,10 +20,10 @@
     "jsx": "react-jsx",
     "sourceMap": true,
```

Comment:
Changed this file to be less development unfriendly, lint still shows that there are things to polish out